app/ptxboa_functions.py

```diff
@@ -12,18 +12,7 @@
 
 def calculate_results_single(api: PtxboaAPI, settings):
     """Calculate results for single country pair."""
-    res = api.calculate(
-        scenario=settings["sel_scenario"],
-        secproc_co2=settings["sel_secproc_co2"],
-        secproc_water=settings["sel_secproc_water"],
-        chain=settings["sel_chain"],
-        res_gen=settings["sel_res_gen_name"],
-        region=settings["sel_region"],
-        country=settings["sel_country_name"],
-        transport=settings["sel_transport"],
-        ship_own_fuel=settings["sel_ship_own_fuel"],
-        output_unit=settings["selOutputUnit"],
-    )
+    res = api.calculate(**settings)
 
     return res
```

Comment:
this function is not used anymore, can we remove it?

Reply:
Maybe we use it later! i would like to keep it